src/home/TabHome.tsx

```diff
@@ -127,31 +127,35 @@ function TabHome(_props: Props) {
     },
     {
       key: 'TransactionFeed',
-      component: showZerionTransactionFeed ? <TransactionFeedV2 /> : <TransactionFeed />,
+      component: <TransactionFeed />,
     },
   ]
 
   const renderItem = ({ item }: { item: any }) => item.component
 
   return (
     <SafeAreaView testID="WalletHome" style={styles.container} edges={[]}>
-      <AnimatedFlatList
-        // Workaround iOS setting an incorrect automatic inset at the top
-        scrollIndicatorInsets={{ top: 0.01 }}
-        scrollEventThrottle={16}
-        refreshControl={refresh}
-        onRefresh={onRefresh}
-        refreshing={isLoading}
-        style={styles.container}
-        contentContainerStyle={{ paddingBottom: insets.bottom }}
-        data={flatListSections}
-        renderItem={renderItem}
-        keyExtractor={keyExtractor}
-        testID="WalletHome/FlatList"
-        // To remove the sticky header entirely remove stickyHeaderIndices & stickyHeaderHiddenOnScroll
-        stickyHeaderIndices={[0]}
-        stickyHeaderHiddenOnScroll={true}
-      />
+      {showZerionTransactionFeed ? (
+        <TransactionFeedV2 />
+      ) : (
```

Comment:
this change is easier to review with "hide white spaces" enabled - i had some layout issues (essentially getting the SectionList of TransactionFeedV2 inside the AnimatedFlatList to take up the remaining space, so that the toast would be displayed at the bottom of the screen for an empty feed) and gave up trying to hack it. it's strange to have a SectionList inside a FlatList, i don't think we need this at all.